src/main/java/org/hyperskill/hstest/common/ReflectionUtils.java

```diff
@@ -252,4 +252,20 @@ public static <T extends StageTest<?>> void setupCwd(T stage) {
 
         FileUtils.chdir(testDir);
     }
+
+    public static List<Class<?>> getTypesAnnotatedWith(String annotationPath) {
```

Comment:
This method:
1. Should be named `getClassesAnnotatedWith`
2. Should not contain `.filter(ReflectionUtils::hasMainMethod)` since classes annotated with `annotationPath` (regarding this utility method) aren't required to have main method.